### PR TITLE
Update etcd workload to use snafu's FIO

### PR DESCRIPTION
**Pull request by rsevilla87**
Replace BYOWL by fio-distributed and use hostPath to improve etcd IO emulation.

### CODE REVIEW

**Comment by jtaleric** [comment]
Nice @rsevilla87 -- any thoughts on how we can keep the pass/fail criteria after the run?

**Comment by jtaleric** [comment]
It isn't really pass / fail, but it gives the CI guys a easy way to see the latency reported : https://github.com/cloud-bulldozer/plow/pull/27/files#diff-5d34d0be0f9fc260c39c4cea6ee46d03L84-L90

**Comment by rsevilla87** [comment]
@jtaleric I've added a threshold variable to be compared with the max 99th latency indexed in ES. Is that you you mean?

File: workloads/etcd-perf/run_etcd_tests_fromgit.sh [hunk 1]
```diff
@@ -42,35 +43,44 @@ cat << EOF | oc create -n my-ripsaw -f -
...
-    server: $_es
-    port: $_es_port
-  clustername: $cloud_name
+    server: ${_es}
+    port: ${_es_port}
+  clustername: ${cloud_name}
   test_user: ${cloud_name}-ci
   metadata_collection: true
   metadata_sa: backpack-view
   metadata_privileged: true
```
Comment (dry923) [inline workloads/etcd-perf/run_etcd_tests_fromgit.sh:56]: Metadata should now be in the form:
metadata:
  collection: true
  serviceaccount: backpack-view
  privileged: true

etc

File: workloads/etcd-perf/run_etcd_tests_fromgit.sh [hunk 2]
```diff
@@ -42,35 +43,44 @@ cat << EOF | oc create -n my-ripsaw -f -
...
+      samples: 5
+      servers: 1
+      jobs:
+        - write
+      bs:
+        - 2300
+      numjobs:
+        - 1
+      filesize: 22Mib
+  global_overrides:
```
Comment (dry923) [inline workloads/etcd-perf/run_etcd_tests_fromgit.sh:72]: I assume this is supposed to be aligned under workload?

File: workloads/etcd-perf/run_etcd_tests_fromgit.sh (hunk 1 above)
Comment (rsevilla87) [reply workloads/etcd-perf/run_etcd_tests_fromgit.sh]: fixed!

File: workloads/etcd-perf/run_etcd_tests_fromgit.sh (hunk 2 above)
Comment (rsevilla87) [reply workloads/etcd-perf/run_etcd_tests_fromgit.sh:73]: fixed!

**Pull request merged**